cmd/flipt/import.go

```diff
@@ -14,6 +14,7 @@ import (
 
 type importCommand struct {
 	dropBeforeImport bool
+	newFlagsOnly     bool
```

Comment:
```suggestion
	insertOnly     bool
```

how about this instead?

Reply:
I would prefer to use insertOnly as more obvious name, but in doubt about convenience of two worded external flag: `--insert-only`
I've made the changes - please have a look. If you'll decide to rename back to `--append` - just write a comment.